src/main/resources/templates/courseInstructor.twig

```diff
@@ -0,0 +1,36 @@
+{% extends 'course.twig' %}
+
+{% block buttons %}
+<div class="course-editor-controls">
+        <button id="editsave" type="button" class="btn btn-primary">Edit</button>
+</div>
+{% endblock %}
+
+{% block script %}
+{{ parent() }}
+<script>
+$(document).ready(function () {
+	$('#editsave').on('click', function (e) {
+		if (grid.editable) {
+		    grid.editable = false;
+		    $('#editsave').html('Edit');
+		    var seats = grid.getSeats();
+		    console.log(seats);
+		    $.ajax({
```

Comment:
nothing on success?

Reply:
No, not for this ajax request. We're just doing a post request to update the seats!